src/components/trashlookup.jsx

```diff
@@ -100,13 +101,11 @@ class TrashLookUp extends Component {
           .add(1, 'd')
           .format(dateFormat);
   }
-
   render() {
     let isAutoTextHidden = this.state.isAutoTextHidden;
     const selectedAddress = _.trim(this.state.selectedAddress);
     const { data } = this.addressData(isAutoTextHidden);
-
-
+    const fullAddress = data.length > 0 ?`${data[0].address1}${`, `} ${data[0].city}${`, `} ${data[0].state}${` `} ${data[0].postalCode}`:'none';
```

Comment:
Can you please create a folder called `Services` and inside that folder move this logic to a file called `AddressService.js`. Inside the file you will want to do something like

```js
const parseAddress = address => {
  return data.length > 0 ?`${data[0].address1}${`, `} ${data[0].city}${`, `} ${data[0].state}${` `} ${data[0].postalCode}`:'none'
}

export {
  parseAddress
}
```

in this file you would then use it this way:

at the top

```js
import { parseAddress } from './Services/AddressService.js'
```

then you can call it like this
```js
const fullAddress = parseAddress(data);
```

In the services folder you should also add a file `AddressService.test.js` and inside that file you should add some tests for your parseAddress function. Once you get to this point, let me know if you need help.

Reply:
services folder is created and called getFullAddress(Address) control